shopify/resources/marketing_event.py

```diff
@@ -0,0 +1,11 @@
+import json
+from ..base import ShopifyResource
+
+class MarketingEvent(ShopifyResource):
+    def add_engagements(self, engagements):
+        """
+        @type engagements: list
+        """
```

Comment:
We aren't using docstrings anywhere else in this project – I think I'd prefer not to add them here now but rather to go back through and add them everywhere if we're going to add them.